roles/remove-node/remove-etcd-node/tasks/main.yml

```diff
@@ -8,7 +8,7 @@
     - inventory_hostname in groups['etcd']
     - ip is not defined
     - access_ip is not defined
-  delegate_to: "{{ groups['etcd']|first }}"
+  delegate_to: "{{ groups['kube-master']|first }}"
```

Comment:
kube-master group has been renamed to kube_control_plane since 486b223e01960e66dc2f2177d739f8b7b8799485